misc/start_faux

```diff
@@ -58,6 +57,9 @@ if [ -n "${options[passwordfail]}" ]; then
     /usr/sbin/sshd -D &
     service xinetd start
     service ssh start
+else
```

Comment:
is stop needed?  Or, if stop is needed, is start needed?

Reply:
Hi Trevor, these were clauses I felt were needed before the refactor to make the code logic work, but after running the tests again with the old start_faux file, I've found I no longer need these. Will remove these on next commit.